examples/test/laser-controls/alignment.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Laser Controls - Alignment</title>
+    <meta name="description" content="Raycaster - A-Frame">
+    <script src="../../../dist/aframe-master.js"></script>
+  </head>
+  <body>
+    <a-scene>
+      <a-entity position="-2 1.5 -2">
+        <a-entity vive-controls cursor="laserEnabled: true" raycaster="objects: null"></a-entity>
+      </a-entity>
+
+      <a-entity position="-1 1.5 -2">
+        <a-entity oculus-touch-controls cursor="laserEnabled: true" raycaster="far: 2; objects: null; origin: 0 0 0.1; direction: 0 -0.4 -1"></a-entity>
+      </a-entity>
+
+      <a-entity position="0 1.5 -2">
+        <a-entity daydream-controls cursor="laserEnabled: true" raycaster="objects: null"></a-entity>
+      </a-entity>
+
+      <a-entity position="1 1.5 -2">
+        <a-entity gearvr-controls cursor="laserEnabled: true" raycaster="objects: null; origin: 0 0.005 0"></a-entity>
```

Comment:
Is there a laser-related reason that `objects:null` is set, or is that just for this demo?